src/Orleans/Streams/PersistentStreams/IStreamQueueBalancer.cs

```diff
@@ -8,27 +8,32 @@ namespace Orleans.Streams
     /// The stream queue balancer is responsible for load balancing queues across all other related queue balancers.  It
     /// notifies any listeners (<code>IStreamQueueBalanceListener</code>) of changes to the distribution of queues.
     /// </summary>
-    internal interface IStreamQueueBalancer
+    public interface IStreamQueueBalancer
     {
         /// <summary>
+        /// Async Initialize logic
```

Comment:
yep, just the doc comments. There are many `Initialize` methods in Orleans, so I wanted them to be aligned. Some examples from our code:
```C#
        /// <summary>
        /// Initializes the external serializer. Called once when the serialization manager creates 
        /// an instance of this type
        /// </summary>
        void Initialize(Logger logger);

    /// <summary>
    /// Initializes the client runtime from the standard client configuration file.
    /// </summary>
    public static void Initialize()

        /// <summary>
        /// Initializes this instance.
        /// </summary>
        public void Initialize()
```

We don't use StyleCop, but standard MS StyleCop rules check that constructor doc comments all follow a pattern like this:
```C#
/// <summary>
/// Initializes a new instance of the <see cref="AssemblyProcessor"/> class.
/// </summary>
```